support_model.py

```diff
@@ -44,6 +45,17 @@ def remove_slack_commands(query):
     return query
 
 
+def get_answer_if_known(query):
+    gc = gspread.service_account(filename='ods-qna-dd4a610895ad.json')
```

Comment:
Where is this file? What does its name mean? Is it a file with credentials?

Reply:
Is it possible to add this info from file to the environment?

Reply:
I'll send it privately. I'm afraid not, it's not possible. Also, change name to ods-qna-secret.json.

Reply:
Ok. I think we can create this file from env text variable. I mean it will be better to make one `env.sh` file with all credentials, tokens and variables. This file of course we will be share it privately.